django/db/backends/oracle/features.py

```diff
@@ -55,3 +55,5 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     """
     supports_callproc_kwargs = True
     supports_over_clause = True
+    # Oracle has limitation of 2^16 - 1 parameters in a query.
```

Comment:
"Oracle is limited to 2^16 - 1 parameters in a query."
or maybe it's better to write `max_query_params = 2^16 − 1` and omit the comment?